### PR TITLE
Math Magicians: refactor with hooks

### PR DESCRIPTION
**Pull request by hamid-murambiwa**
In this project I refactored the Calculator component in the calcHooks component. Replaced the class based components, to use hooks. Made the following changes :
- Refactor the Calculator component in the calcHooks.js component
- Change class based component to a functional component with hooks in the calcHooks.js file
- Change class based component to a functional component with hooks in the App.js file

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @hamid-murambiwa 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,10 @@
-import calculator from './components/Calculator';
-import React from 'react';
-class App extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = new calculator;
-    }
+import CalcHooks from './components/calcHooks.js';
+// import React from 'react';
```
Comment (mikenath223) [inline src/App.js]: - [ ] Please remove this commented code.

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.15.0",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "big.js": "^6.1.1",
```
Comment (mikenath223) [inline package.json:9]: - [ ] In `.eslintrc.json` you need to use the exact same rules listed [here](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json).

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## Thank you for implementing all the project requirements, you've done a great job :trophy: There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**